javascript/packages/orchestrator/src/test-runner/assertions.ts

```diff
@@ -89,34 +89,22 @@ const CalcMetrics = ({
   timeout,
 }: FnArgs) => {
   const comparatorFn = comparators[op!];
-  const mathFn = (a: number, b: number): number => {
-    return math_ops === "Minus" ? a - b : a + b;
-  };
 
   return async (network: Network) => {
     const nodes = network.getNodes(node_name!);
-    const promise_a = nodes.map((node: any) =>
-      node.getMetric(
-        metric_name_a!,
-        toChaiComparator(op!),
-        target_value!,
-        timeout || DEFAULT_INDIVIDUAL_TEST_TIMEOUT,
-      ),
-    );
-
-    const promise_b = nodes.map((node: any) =>
-      node.getMetric(
-        metric_name_b!,
-        toChaiComparator(op!),
-        target_value!,
-        timeout || DEFAULT_INDIVIDUAL_TEST_TIMEOUT,
+    const results = await Promise.all(
+      nodes.map((node: any) =>
+        node.getCalcMetric(
+          metric_name_a,
+          metric_name_b,
+          math_ops,
```

Comment:
I think we should avoid mixing camelCase and snake_case in the code and use a standard